kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ManageLessonModals.vue

```diff
@@ -115,8 +115,8 @@
         const { id } = this.currentLesson;
         return LessonResource.deleteModel({ id })
           .then(() => {
-            this.$router.replace(
-              this.$router.getRoute('PLAN_LESSONS_ROOT', { classId: this.classId }),
+            this.$router.replace(  
+              this.$router.getRoute('PLAN_LESSONS_ROOT'),
```

Comment:
thank you for your contribution, @ShivangRawat30 - we appreciate your work on this!

i believe a different approach to solving this problem will have to be explored. the code removed by this PR on line 119, `{ classId: this.classId })`, was only recently added to our codebase and it serves a specific and important purpose. this code shares the relevant `classId` with the `PLAN_LESSONS_ROOT` page, and the page needs that information to render correctly.  without knowing `classId`, `PLAN_LESSONS_ROOT` has no awareness of which list of lessons to show the user, as lessons are tied to a specific `classId`. 

since that would result in broken functionality, we aren't able to incorporate this suggested change, but if you're interested in continuing to work on it, i think it's worth exploring alternate solutions! :)

Reply:
I'm keen on persisting with the project and exploring alternative solutions.

Reply:
excellent - please feel free to re-request my review when you have an update!